tests/Unit/API/Site/Controllers/MerchantCenter/RequestReviewControllerTest.php

```diff
@@ -116,6 +119,87 @@ public function test_merged_response() {
 		], $response->get_data() );
 	}
 
+	public function test_no_offers_reponse() {
+
+		$this->middleware->expects( $this->once() )
+		                 ->method( 'get_account_review_status' )
+		                 ->willReturn(
+			                 [
+				                 'programTypeA' => [
+					                 'status' => 200,
+					                 'data'   => [
+						                 "globalState" =>  "ENABLED",
+						                 'regionStatuses' => [
+							                 [
+								                 'reviewEligibilityStatus' => 'INELIGIBLE',
+								                 'eligibilityStatus'       => 'APPROVED',
+							                 ],
+							                 [
+								                 'reviewEligibilityStatus' => 'ELIGIBLE',
+								                 'eligibilityStatus'       => 'DISAPPROVED',
+								                 'reviewIssues'            => [ 'one' ]
+							                 ],
+						                 ]
+					                 ]
+				                 ],
+				                 'programTypeB' => [
+					                 'status' => 200,
+					                 'data'   => [
+						                 "globalState" =>  "NO_OFFERS_UPLOADED"
```

Comment:
❓

Isn't it better to use the constants `RequestReviewStatuses::NO_OFFERS` etc instead of doing this manually?